assets/product-modal.js

```diff
@@ -0,0 +1,34 @@
+if (!customElements.get('product-modal')) {
```

Comment:
Nothing changed on this file. Just moved it from `featured-product` and `main-product`.